docs/src/main/sphinx/installation/plugins.md

```diff
@@ -0,0 +1,311 @@
+# Plugins 
+
+Trino uses a plugin architecture to extend its capabilities and integrate with
+various data sources and other systems. Trino includes many plugins as part of
+the binary packages - specifically the `tar.gz` archive and the Docker
+container.
```

Comment:
container -> image

Usually "container" would refer to a running instance of the image